### PR TITLE
[WIP] [CI:BUILD] Add Podman-in-Podman GitLab runner image

### PR DESCRIPTION
**Pull request by cevich**
***FIXME***: Add a real commit message.

***FIXME***: I'm not sure if the podman-repo is the appropriate place for this.

***FIXME***: This builds (including multi-arch) but is mostly untested as-is.  I am successfully running a previous version (privately) but that may just be my good-luck :smile: 

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by cevich** [comment]
force-push: Changed the `--privileged` default option to ***FALSE*** instead of true.  Add some other overrides as well, all else remains equal.

File: contrib/pipglr/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+the configuration.  If your intent is to simply update or modify the
+configuration, please edit the config.toml file within the
+`gitlab-runner-config` volume.
+
+Note: These commands assume you have both `podman` and `jq` available.
+
+```bash
+$ echo '<registration token>' | podman secret create REGISTRATION_TOKEN -
+$ export IMAGE=<image FQIN:TAG>
+$ eval $(podman inspect --format=json $IMAGE | jq -r .[].Labels.register)
```
Comment (lsm5) [inline contrib/pipglr/README.md:37]: Is the command right?
```
$ eval $(podman inspect --format=json $IMAGE | jq -r .[].Labels.register)
-bash: null: command not found

$ echo $IMAGE
registry.gitlab.com/rhcontainerbot/pkg-builder:fedora
```

Comment (lsm5) [reply contrib/pipglr/README.md:37]: @cevich ping

Comment (cevich) [reply contrib/pipglr/README.md:37]: The eval command is intended to be used on hosts that don't (yet) support `podman container runlabel` (like CentOS 9 IIRC).  In any case, that's an odd error, it suggests that the image never had the labels added to it, let me check the `Containerfile`...

Comment (cevich) [reply contrib/pipglr/README.md:37]: ...I copied these details over to gitlab (so I can have automated image builds right now).   This is what I did to get a test-runner up and running for that repo:

```bash
$ echo '<TOKEN>' | podman secret create REGISTRATION_TOKEN -
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel register registry.gitlab.com/qontainers/pipglr:latest
$ podman container runlabel run registry.gitlab.com/qontainers/pipglr:latest
```

Then I check the CI/CD Settings page on gitlab, under runners section, it shows a green dot.

Comment (cevich) [reply contrib/pipglr/README.md:48]: Updated docs.

**Comment by cevich** [comment]
I'm going to abandon this for now in favor of https://gitlab.com/qontainers/pipglr because I cannot get our multiarch builds running.  It seems there aren't binaries available and/or they're named with oddball non-standard names.  Can re-open this and https://github.com/containers/automation_images/pull/233 if needs and/or situation changes.

**Pull request closed without merging**